apps/TestDriver.cpp

```diff
@@ -7,14 +7,36 @@
 #include "HDK.h"
 
 #include <iostream>
-#include <memory>
+
+#include "Shared/ArrowUtil.h"
+
+#include <arrow/api.h>
 
 int main(void) {
-  int table_id = 0;
+  std::cout << "Hello, world" << std::endl;
+
+  // Load some data
+  auto col_x = std::make_shared<arrow::Field>("x", arrow::int32());
```

Comment:
I wanted to show how you could provide pointers, though I guess it's really arrow arrays and not so different from the CSV example. 

As far as the API goes, does it make sense to mix HDK types and "Arrow" types in the above example? Since Arrow is the CSV reader, shouldn't we use Arrow types and then late the conversion to HDK happen during import? I guess it doesn't matter, just feels a bit odd semantically.

Reply:
If you wanted to demonstrate Arrow table import then my example surely doesn't fit.

I don't see any odd types mix. Arrow API uses Arrow types, HDK API uses HDK types. That looks quite natural.